membership-attribute-service/app/models/FeastApp.scala

```diff
@@ -17,8 +17,7 @@ object FeastApp {
   private def isBeforeFeastLaunch(dt: LocalDate): Boolean = dt.isBefore(FeastIosLaunchDate)
 
   def shouldGetFeastAccess(attributes: Attributes): Boolean =
-    attributes.isStaffTier ||
```

Comment:
this was misleading - staff who get digi access will get feast via `attributes.digitalSubscriberHasActivePlan` below (although in a convoluted way - this logic needs refactoring)